test/extensions/filters/http/kill_request/kill_request_filter_test.cc

```diff
@@ -209,6 +209,24 @@ TEST_F(KillRequestFilterTest, KillsBasedOnDirection) {
                "KillRequestFilter is crashing Envoy!!!");
 }
 
+TEST_F(KillRequestFilterTest, PerRouteKillSettingFound) {
+  envoy::extensions::filters::http::kill_request::v3::KillRequest kill_request;
+  setUpTest(kill_request);
+  request_headers_.addCopy("x-envoy-kill-request", "true");
+
+  envoy::extensions::filters::http::kill_request::v3::KillRequest route_level_kill_request;
+  // Set probability to zero to make isKillRequestEnabled return false
+  route_level_kill_request.mutable_probability()->set_numerator(0);
+  route_level_kill_request.set_kill_request_header("x-custom-kill-request");
+
+  // Return valid kill setting on the REQUEST direction
+  KillSettings const kill_settings(route_level_kill_request);
```

Comment:
Really tiny nit: 
From style guide: https://google.github.io/styleguide/cppguide.html

Putting the const first is arguably more readable, since it follows English in putting the "adjective" (const) before the "noun" (int).


Most of the times in Envoy I see `const Foo foo` vs `Foo const foo`

Reply:
done